.env.template

```diff
@@ -4,3 +4,9 @@
 
 TWILIO_ACCOUNT_SID=YourAccountSIDHere
 LOCAL_AUTH_TOKEN=YourAuthTokenHere
+
+# Or use your own STUN/TURN server or just use Google's: stun.l.google.com:19302,stun1.l.google.com:19302,stun2.l.google.com:19302,stun3.l.google.com:19302,stun4.l.google.com:19302,stun01.sipphone.com,stun.ekiga.net,stun.fwdnet.net,stun.ideasip.com,stun.iptel.org,stun.rixtelecom.se,stun.schlund.de,stunserver.org,stun.softjoys.com,stun.voiparound.com,stun.voipbuster.com,stun.voipstunt.com,stun.voxgratia.org,stun.xten.com
+ICE_SERVER_URLS=
```

Comment:
You should maybe add an example config here, something like <server1>,<server2> or just say that it's a comma-separated list.

Also, multiple servers but only one set of credentials? Does that make sense?

Reply:
There is an example in line 11 now.

No, it does not really make sense.

Reply:
One ICE server including `ICE_SERVER_URLS`, `ICE_SERVER_USERNAME `, `ICE_SERVER_CREDENTIAL ` and `ICE_SERVER_CREDENTIALTYPE `.
And one ICE server could have multiple URLs with one credential.

[RTCIceServer](https://developer.mozilla.org/en-US/docs/Web/API/RTCIceServer)

Reply:
ahh, I see!